src/backend/aspen/database/models/sample.py

```diff
@@ -242,6 +243,12 @@ class Sample(idbase, DictMixin):  # type: ignore
         ),
     )
 
+    accessions = relationship(
+        Accessions,
+        backref=backref("samples", cascade="all, delete", uselist=False),
```

Comment:
I think the backref should just be 'sample' here since an accession will only be associated with a single sample.

Reply:
`"samples"` as in the `samples` table. The `uselist=False` tells SQLAlchemy that this is a one-to-one relationship

Reply:
Or maybe you're right? The SQLAlchemy docs are so worthless, I can't tell what the hell that argument refers to

Reply:
nvm what I wrote earlier, changed it